ansible_collections/arista/avd/roles/eos_designs/python_modules/base/base.py

```diff
@@ -0,0 +1,664 @@
+
+    @cached_property
+    def event_monitor(self):
+        """
+        event_monitor set based on event_monitor data-model
+        """
+        if get(self._hostvars, 'event_monitor') is True:
+            return {'enabled': 'true'}
+        return None
+
+    @cached_property
+    def event_handlers(self):
+        """
+        event_handlers set based on event_handlers data-model
+        """
+        return get(self._hostvars, 'event_handlers')
+
+    @cached_property
+    def load_interval(self):
+        """
+        load_interval set based on load_interval_default variable
+        """
+        if (load_interval_default := get(self._hostvars, 'load_interval_default')) is not None:
+            return {'default': load_interval_default}
+        return None
+
+    @cached_property
+    def queue_monitor_length(self):
+        """
+        queue_monitor_length set based on queue_monitor_length data-model and
+        switch.feature_support.queue_monitor_length_notify fact
+        """
+        if (queue_monitor_length := get(self._hostvars, 'queue_monitor_length')) is None:
+            return None
+
+        queue_monitor_length_dict = {'enabled': True}
+        queue_monitor_length_notifying = get(queue_monitor_length, 'notifying')
+        notify_supported = get(self._platform_settings, "feature_support.queue_monitor_length_notify")
+        if (
+                queue_monitor_length_notifying is not None and
+                notify_supported is not False
+        ):
+            queue_monitor_length_dict['notifying'] = queue_monitor_length_notifying
+
+        if get(queue_monitor_length, 'log') is not None:
+            queue_monitor_length_dict['log'] = queue_monitor_length.get('log')
+
+        return queue_monitor_length_dict
+
+    @cached_property
+    def name_server(self):
+        """
+        name_server set based on name_servers data-model and mgmt_interface_vrf
+        """
+        if (name_servers := get(self._hostvars, 'name_servers')) is not None:
+            return {
+                'source':
+                    {
+                        'vrf': self._mgmt_interface_vrf
+                    },
+                'nodes': name_servers
+            }
+
+        return None
+
+    @cached_property
+    def redundancy(self):
+        """
+        redundancy set based on redundancy data-model
+        """
+        if get(self._hostvars, 'redundancy') is not None:
+            return {'protocol': get(self._hostvars, 'redundancy.protocol')}
+        return None
+
+    @cached_property
+    def snmp_server(self):
+        """
+        snmp_server set based on snmp_settings data-model, using various snmp_settings information.
+
+        if snmp_settings.compute_local_engineid is True we will use sha1 to create a
+        unique local_engine_id value based on switch.hostname and switch.mgmt_ip facts.
+
+        If user.version is set to 'v3', compute_local_engineid and compute_v3_user_localized_key are set to 'True'
```

Comment:
either the documentation or the template is incorrect.  the template uses `compute_v3_user_localized_key` but the document references `compute_v3_localized_key`

[management settings doc](https://github.com/aristanetworks/ansible-avd/blob/58d3a1e61a0715951f0d9a490b38c6601c4cb040/ansible_collections/arista/avd/roles/eos_designs/doc/management-settings.md?plain=1#L84)

Reply:
@gmuloc, which one should we honor, the template or the document?

Reply:
let's use the template key, so we need to update the docs.

Reply:
@carlbuchmann will submit a separate PR on the doc fix